spring-kafka/src/main/java/org/springframework/kafka/support/converter/AbstractJavaTypeMapper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 the original author or authors.
+ * Copyright 2018 the original author or authors.
```

Comment:
Well, you know we have to supply here a range of years with the current in the end.